pavelib/assets.py

```diff
@@ -292,6 +301,28 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
+    """
+    Process vendor libraries installed via NPM.
+    """
+    # Skip processing of the libraries if this is just a dry run
+    if tasks.environment.dry_run:
+        tasks.environment.info("install npm_assets")
+        return
+
+    # Ensure that the vendor directory exists
+    sh('mkdir -p {vendor_dir}'.format(
```

Comment:
I realize paver is inconsistent on this, but the best practice is to use a python library to accomplish this. `mkdir_p` is called a few times in this file so I'd use that.